doc_admin/AdminManual.adoc

```diff
@@ -85,8 +85,14 @@ For the Open Hospital installation on every client, it is sufficient to copy and
 
 For the Open Hospital configuration refer to the chapter <<config-files,Configuration Files>>.
 
+==== 3.1.1 Prerequisites - Software Compatibility matrix
+
+Before starting, checke the following software compatibility matrix, in order to verify the external software versions needed to run Open Hospital and the current available extended features for every version, architecture and Operating System.
```

Comment:
Here "Operating System" is upper case; line 97 it is all lower case; I would pick the lower case version

Reply:
Agreed ! corrected